{"uid":"d481b1f267b56997","name":"Login with invalid credentials test","fullName":"tests.login_test.TestLogin#test_invalid_login","historyId":"9bbc2729137fa409b9bc8067987e1d0d","time":{"start":1705281413841,"stop":1705281424171,"duration":10330},"description":"invalid login","descriptionHtml":"<p>invalid login</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55c4cd846f83 <unknown>\n#1 0x55c4cd4ffcf7 <unknown>\n#2 0x55c4cd54f99e <unknown>\n#3 0x55c4cd54faa1 <unknown>\n#4 0x55c4cd59ad64 <unknown>\n#5 0x55c4cd5790dd <unknown>\n#6 0x55c4cd598006 <unknown>\n#7 0x55c4cd578e53 <unknown>\n#8 0x55c4cd540dd4 <unknown>\n#9 0x55c4cd5421de <unknown>\n#10 0x55c4cd80b531 <unknown>\n#11 0x55c4cd80f455 <unknown>\n#12 0x55c4cd7f7f55 <unknown>\n#13 0x55c4cd8100ef <unknown>\n#14 0x55c4cd7db99f <unknown>\n#15 0x55c4cd834008 <unknown>\n#16 0x55c4cd8341d7 <unknown>\n#17 0x55c4cd846124 <unknown>\n#18 0x7fade9694ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7fdc25b4cec0>\nemail = 'elias@gmail.com', password = '12345Tr'\ndata = Data(forgot_password=ForgotPassword(error_message=\"We can't find a user with that e-mail address.\", success_message='A...iteria', non_existing_project='Non Existing', project_name='test project', project_type='quiz', template_type='Blank'))\n\n    @allure.description(\"invalid login\")\n    @allure.title(\"Login with invalid credentials test\")\n    @pytest.mark.parametrize(\"email, password\", users)\n    @pytest.mark.run(order=3)\n    def test_invalid_login(self, email: str, password: str, data: Data):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:60: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"6ae0d5a91d524e2e0bc53a22cfffb6f2\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7fdc25b37c40>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x55c4cd846f83 <unknown>\nE       #1 0x55c4cd4ffcf7 <unknown>\nE       #2 0x55c4cd54f99e <unknown>\nE       #3 0x55c4cd54faa1 <unknown>\nE       #4 0x55c4cd59ad64 <unknown>\nE       #5 0x55c4cd5790dd <unknown>\nE       #6 0x55c4cd598006 <unknown>\nE       #7 0x55c4cd578e53 <unknown>\nE       #8 0x55c4cd540dd4 <unknown>\nE       #9 0x55c4cd5421de <unknown>\nE       #10 0x55c4cd80b531 <unknown>\nE       #11 0x55c4cd80f455 <unknown>\nE       #12 0x55c4cd7f7f55 <unknown>\nE       #13 0x55c4cd8100ef <unknown>\nE       #14 0x55c4cd7db99f <unknown>\nE       #15 0x55c4cd834008 <unknown>\nE       #16 0x55c4cd8341d7 <unknown>\nE       #17 0x55c4cd846124 <unknown>\nE       #18 0x7fade9694ac3 <unknown>\n\n.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_verify_url","time":{"start":1705281364488,"stop":1705281364489,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"base_url","time":{"start":1705281364488,"stop":1705281364488,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"data","time":{"start":1705281364489,"stop":1705281364489,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"invalid login","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55c4cd846f83 <unknown>\n#1 0x55c4cd4ffcf7 <unknown>\n#2 0x55c4cd54f99e <unknown>\n#3 0x55c4cd54faa1 <unknown>\n#4 0x55c4cd59ad64 <unknown>\n#5 0x55c4cd5790dd <unknown>\n#6 0x55c4cd598006 <unknown>\n#7 0x55c4cd578e53 <unknown>\n#8 0x55c4cd540dd4 <unknown>\n#9 0x55c4cd5421de <unknown>\n#10 0x55c4cd80b531 <unknown>\n#11 0x55c4cd80f455 <unknown>\n#12 0x55c4cd7f7f55 <unknown>\n#13 0x55c4cd8100ef <unknown>\n#14 0x55c4cd7db99f <unknown>\n#15 0x55c4cd834008 <unknown>\n#16 0x55c4cd8341d7 <unknown>\n#17 0x55c4cd846124 <unknown>\n#18 0x7fade9694ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7fdc25b4cec0>\nemail = 'elias@gmail.com', password = '12345Tr'\ndata = Data(forgot_password=ForgotPassword(error_message=\"We can't find a user with that e-mail address.\", success_message='A...iteria', non_existing_project='Non Existing', project_name='test project', project_type='quiz', template_type='Blank'))\n\n    @allure.description(\"invalid login\")\n    @allure.title(\"Login with invalid credentials test\")\n    @pytest.mark.parametrize(\"email, password\", users)\n    @pytest.mark.run(order=3)\n    def test_invalid_login(self, email: str, password: str, data: Data):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:60: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"6ae0d5a91d524e2e0bc53a22cfffb6f2\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7fdc25b37c40>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x55c4cd846f83 <unknown>\nE       #1 0x55c4cd4ffcf7 <unknown>\nE       #2 0x55c4cd54f99e <unknown>\nE       #3 0x55c4cd54faa1 <unknown>\nE       #4 0x55c4cd59ad64 <unknown>\nE       #5 0x55c4cd5790dd <unknown>\nE       #6 0x55c4cd598006 <unknown>\nE       #7 0x55c4cd578e53 <unknown>\nE       #8 0x55c4cd540dd4 <unknown>\nE       #9 0x55c4cd5421de <unknown>\nE       #10 0x55c4cd80b531 <unknown>\nE       #11 0x55c4cd80f455 <unknown>\nE       #12 0x55c4cd7f7f55 <unknown>\nE       #13 0x55c4cd8100ef <unknown>\nE       #14 0x55c4cd7db99f <unknown>\nE       #15 0x55c4cd834008 <unknown>\nE       #16 0x55c4cd8341d7 <unknown>\nE       #17 0x55c4cd846124 <unknown>\nE       #18 0x7fade9694ac3 <unknown>\n\n.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","steps":[{"name":"Click Login link","time":{"start":1705281413842,"stop":1705281424171,"duration":10329},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55c4cd846f83 <unknown>\n#1 0x55c4cd4ffcf7 <unknown>\n#2 0x55c4cd54f99e <unknown>\n#3 0x55c4cd54faa1 <unknown>\n#4 0x55c4cd59ad64 <unknown>\n#5 0x55c4cd5790dd <unknown>\n#6 0x55c4cd598006 <unknown>\n#7 0x55c4cd578e53 <unknown>\n#8 0x55c4cd540dd4 <unknown>\n#9 0x55c4cd5421de <unknown>\n#10 0x55c4cd80b531 <unknown>\n#11 0x55c4cd80f455 <unknown>\n#12 0x55c4cd7f7f55 <unknown>\n#13 0x55c4cd8100ef <unknown>\n#14 0x55c4cd7db99f <unknown>\n#15 0x55c4cd834008 <unknown>\n#16 0x55c4cd8341d7 <unknown>\n#17 0x55c4cd846124 <unknown>\n#18 0x7fade9694ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/base_page.py\", line 60, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"f041925b94e08cbf","name":"public ip address","source":"f041925b94e08cbf.txt","type":"text/plain","size":15},{"uid":"4468165ae79eb312","name":"Cookies","source":"4468165ae79eb312.json","type":"application/json","size":4465},{"uid":"c2f07b33f866997b","name":"Session Storage","source":"c2f07b33f866997b.json","type":"application/json","size":359},{"uid":"d84e3248f4d66557","name":"Local Storage","source":"d84e3248f4d66557.json","type":"application/json","size":208},{"uid":"20bc904179b265f9","name":"Console Logs","source":"20bc904179b265f9.json","type":"application/json","size":2},{"uid":"46b20825ca7a55da","name":"Full Page Screenshot","source":"46b20825ca7a55da.png","type":"image/png","size":5726424},{"uid":"5f14aa56c13e9b6c","name":"URL","source":"5f14aa56c13e9b6c.uri","type":"text/uri-list","size":23},{"uid":"b4b9e02eeda0f91f","name":"Network Logs","source":"b4b9e02eeda0f91f.json","type":"application/json","size":916088}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":8,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"severity","value":"blocker"},{"name":"feature","value":"Login"},{"name":"epic","value":"Security"},{"name":"tag","value":"run(order=3)"},{"name":"tag","value":"security"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"login_test"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az1200-785"},{"name":"thread","value":"2312-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.login_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"email","value":"'elias@gmail.com'"},{"name":"password","value":"'12345Tr'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"blocker","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["run(order=3)","security"]},"source":"d481b1f267b56997.json","parameterValues":["'elias@gmail.com'","'12345Tr'"]}